app/models/gsa18f/proposal_form.rb

```diff
@@ -0,0 +1,87 @@
+module Gsa18f
+  DATA = YAML.load_file("#{Rails.root}/config/data/18f.yaml")
+
+  class ProposalForm
+    include SimpleFormObject
+
+    URGENCY = DATA['URGENCY']
+    OFFICES = DATA['OFFICES']
+    
+    attribute :requester, :user
+    attribute :office, :string
+    attribute :justification, :text
+    attribute :link_to_product, :string
+    attribute :quantity, :integer
+    attribute :date_requested, :datetime
+    attribute :urgency, :string
```

Comment:
I thought I left a comment about it before, but I think we should store the urgency as an int.